### PR TITLE
Tickets/dm39496: Configured reports to run on BTS AND TTS

### PR DESCRIPTION
**Pull request by rbovill**
* Fixed a bug in the verify_software_versions keyword.
* Renamed all ComCam test suites to BigCamera.
* Parameterized the test stand specific Camera values.
* Moved the OCPS 2/3 test cases to the BigCamera suites.
* General cleanup.

### CODE REVIEW

File: configure_python_interpreter.txt [hunk 1]
```diff
@@ -18,7 +18,7 @@ for csc in cscs:
 
 
 from QueryEfd import QueryEfd
-efd = QueryEfd("7.0.0", "12.0.0", "6.10.4p2")
+efd = QueryEfd("7.4.0", "1280.0", "6.10.4p2", "tucson_teststand_efd")
```
Comment (mareuter) [inline configure_python_interpreter.txt:21]: What's the 1280.0?  Doesn't look like a version number.

File: Offline/Verify_Camera_Offline.robot [hunk 2]
```diff
@@ -46,19 +46,19 @@ Verify ATCamera OfflineDetailedStates timing
     Verify Time Delta    ATCamera    logevent_summaryState    ${offdet_topic}    ${time_window}
```
Comment (mareuter) [inline Offline/Verify_Camera_Offline.robot:46]: Why isn't this called `Verify_BigCamera_Offline.robot`?

**Review by mareuter — APPROVED** [review]
I'd like for OCPS:1 to move to the LATISS tests from ObsSys in order to match what's happening with the big cameras. Other than that, there are a few comments to look over.

File: Offline/Verify_Camera_Offline.robot (hunk 2 above)
Comment (rbovill) [reply Offline/Verify_Camera_Offline.robot:46]: Because it tests both ATCamera and the BigCamera. Renamed to Verify_Cameras_Offline.robot for clarity.

**Pull request merged**